CMakeLists.txt

```diff
@@ -882,6 +887,24 @@ if(WITH_OPTIM)
             add_feature_info(XSAVE 1 "Support XSAVE intrinsics using \"${XSAVEFLAG}\"")
             set_property(SOURCE ${ARCHDIR}/x86_features.c PROPERTY COMPILE_FLAGS "${XSAVEFLAG}")
         endif()
+    elseif(BASEARCH_RISCV_FOUND)
```

Comment:
This `elseif` should go above `elseif` for `BASEARCH_S360_FOUND`, because it is sorted alphabetically.

Reply:
Thanks, it's done.